### DISCUSSION TITLE
Multiple GPU with nvenc in one app instance

### DISCUSSION THREAD

**Discussion by k0stett** [datarhei/restreamer#546] [General]
Is it possible to use two GPUs with nvenc in one instance of Restreamer? Nvidia Container Runtime allows to run container with parameter `-e NVIDIA_VISIBLE_DEVICES=0,1` but idk will Restreamer use second GPU or not.

**Comment by svenerbeck** [comment]
Hello @k0stett 
You will need to test this out yourself. We would appreciate it if you could share the results of your tests. Thank you and best regards from datarhei.

Cheerio
Sven
